x-pack/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -204,7 +204,7 @@ export function ProcessTreeNode({
   const shouldRenderChildren = childrenExpanded && children?.length > 0;
   const childrenTreeDepth = depth + 1;
 
-  const showUserEscalation = user.id !== parent.user.id;
+  const showUserEscalation = user.id && user.id !== parent.user?.id;
```

Comment:
User was undefined sometimes with the range of commits I built the endpoint on, so needed this change.